File: Aulas/aula_04/triangulo.cpp
#include <iostream>
#include <iomanip>
using namespace std;

class Triangulo {
    public 
    double b = 0, h = 0
    double calc_area(){
        return b * h / 2
    }
}

int main() {
    Triangulo = x; //variavel não precisa ser instanciada, exemplo em java que necessita ser instanciada Triangulo x = new Triangulo()
    cout << x.b << " " << x.h << endl;
    cout << "informe o valor da base:";
    cin >> x.b
    cout <<"informe o valor da altura:"
    cin >> x.h;
    cout << fixed << setprecision(2);
    cout << "Área =" << x.calc_area() << endl;
}

File: Aulas/aula_04/triangulo.py
# objetos == variáveis que carregam o nome da classe

class Triangulo:
    def __init__(self):
        self.b = 0
        self.h = 0
    def calc_area(self):
        return self.b * self.h / 2

x = Triangulo()
x.b = float(input("informe a base"))
x.h = float(input())

print(f"Area = {x.b*x.h}")